packages/saved-views-react/src/applySavedView.ts

```diff
@@ -74,11 +75,18 @@ export interface ApplySavedViewSettings {
   perModelCategoryVisibility?: ApplyStrategy | "clear" | undefined;
 
   /**
-   * How to handle visibility of models and categories that exist in iModel but
-   * are not captured in Saved View data.
-   * @default "hidden"
+   * How to handle the visibility of models that exist in iModel,
+   * including those not captured in Saved View data.
+   * @default '{ enabled: "ignore", disabled: "ignore", other: "ignore" }'
    */
-  modelAndCategoryVisibilityFallback?: "visible" | "hidden" | undefined;
+  models?: ElementSettings | undefined;
```

Comment:
I would prefer a different name than `ElementSettings` since we are not dealing with "elements" in the BIS sense. What about just `ApplyVisibilitySettings` or something else?